Automattic/vip-go-mu-plugins
# Update JavaScript dependencies

## Description

This PR takes care of Dependabot alerts and updates the JS dependencies to their latest versions.

## Changelog Description

### Developer Tools

 * Update eslines from 1.1.0 to 2.0.0
 * Update eslint from 5.16.0 to 7.32.0

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test

1. Check out PR.
2. Create a dummy commit with PHP files violating our code standards
3. Try to push it; observe that pre-commit/pre-push hooks work by reporting violations.
